alicloud/data_source_alicloud_ssl_vpn_client_certs_test.go

```diff
@@ -0,0 +1,120 @@
+package alicloud
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAlicloudSslVpnClientCertsDataSource_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccPreCheckWithAccountSiteType(t, IntlSite)
+		},
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckAlicloudSslVpnClientCertsDataCfg,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAlicloudDataSourceID("data.alicloud_ssl_vpn_client_certs.foo"),
+					resource.TestCheckResourceAttr("data.alicloud_ssl_vpn_client_certs.foo", "ids.#", "1"),
+					resource.TestCheckResourceAttr("data.alicloud_ssl_vpn_client_certs.foo", "names.#", "1"),
+					resource.TestCheckResourceAttr("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.#", "1"),
+					resource.TestCheckResourceAttr("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.0.name", "tf-testAccSslVpnClientCertsDataResource"),
+					resource.TestCheckResourceAttrSet("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.0.ssl_vpn_server_id"),
+					resource.TestCheckResourceAttrSet("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.0.create_time"),
+					resource.TestCheckResourceAttrSet("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.0.ssl_vpn_client_cert_id"),
+					resource.TestCheckResourceAttrSet("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.0.end_time"),
+					resource.TestCheckResourceAttrSet("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.0.status"),
+					resource.TestCheckResourceAttrSet("data.alicloud_ssl_vpn_client_certs.foo", "ssl_vpn_client_certs.0.region_id"),
+				),
```

Comment:
please add other steps to test the scenario that a fake parameter can not found the resource  and using a sole  parameter to find the specified resource . You can refer to : https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/data_source_alicloud_ddoscoo_instances_test.go